.env.gitpod

```diff
@@ -27,7 +27,7 @@ MAXMIND_USER_ID = 123456
 MAXMIND_LICENCE_KEY = "xxxxxxxxxxxxxxxx"
 
 MICROSOFT_PROVIDER_ID = 1234
-MICROSOFT_LOCAL_FILE_DIRECTORY = "https://tgwf-web-app-test.s3.nl-ams.scw.cloud/data-imports/ms-azure-ip-ranges-2022-04-25.json"
+MICROSOFT_REMOTE_API_ENDPOINT = "https://tgwf-web-app-test.s3.nl-ams.scw.cloud/data-imports/ms-azure-ip-ranges-2022-04-25.json"
```

Comment:
We seatled with this structure because we made requests to this url in the importer (the constant is only used once).
We can improve this by using the `object_storage` class and calling the bucket from there. We would only need to set this constant as:
`MICROSOFT_REMOTE_API_ENDPOINT = "data-imports/ms-azure-ip-ranges-"`
(As I suggested on one of my other comments, the naming could be improved)

With this prefix we can do a filter function on the bucket to get the latest version of that file, which already dynamically changes its name based on when it is lastly updated.
By doing this, the constant can easily be used by more files as this prefix is also needed elsewhere.